diff --git a/configure.ac b/configure.ac
index 473cea5..2da2c5d 100644
--- a/configure.ac
+++ b/configure.ac
@@ -831,16 +831,7 @@ AC_CHECK_FUNCS([freeaddrinfo getnameinfo fork writev getgrouplist])
 
 AC_SEARCH_LIBS(basename, gen, AC_DEFINE(HAVE_BASENAME))
 
-# Solaris needs ptmx
-if test -z "$no_ptmx_check" ; then
-	if test x"$cross_compiling" = x"no" ; then
-		if test -e /dev/ptmx ; then
-			AC_DEFINE(USE_DEV_PTMX,1,Use /dev/ptmx)
-		fi
-	else
-		AC_MSG_NOTICE([Not checking for /dev/ptmx, we're cross-compiling])
-	fi
-fi
+AC_DEFINE(USE_DEV_PTMX,1,Use /dev/ptmx)
 
 if test -z "$no_ptc_check" ; then
 	if test x"$cross_compiling" = x"no" ; then
